### PR TITLE
par: test added

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/par.rb [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-  url 'http://www.nicemice.net/par/Par152.tar.gz'
-  version '1.52'
-  sha1 '4b83d2ec593bb45ee46d4b7c2bfc590e1f4a41a8'
+  homepage "http://www.nicemice.net/par/"
+  url "http://www.nicemice.net/par/Par152.tar.gz"
+  version "1.52"
+  sha1 "4b83d2ec593bb45ee46d4b7c2bfc590e1f4a41a8"
 
   # A patch by Jérôme Pouiller that adds support for multibyte
   # charsets (like UTF-8), plus Debian packaging.
```
Comment (mikemcquaid) [inline Library/Formula/par.rb]: Can you submit this upstream? No worries if not.

File: Library/Formula/par.rb [hunk 2]
```diff
@@ -15,8 +13,13 @@ class Par < Formula
   end
 
   def install
-    system "make -f protoMakefile"
+    system "make", "-f", "protoMakefile"
```
Comment (mikemcquaid) [inline Library/Formula/par.rb]: Is there a `make install`?

Comment (bfontaine) [inline Library/Formula/par.rb]: Nope.

File: Library/Formula/par.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/par.rb]: Upstream is aware of this patch, it’s linked on their homepage. I don’t know why they haven’t applied it yet.

**Pull request closed without merging**